human_trajectory/scripts/trajectory_publisher.py

```diff
@@ -71,9 +77,31 @@ def _publish_online_data(self, incremental=False):
                     del self.trajs.traj[uuid]
                     del self.trajs.complete[uuid]
 
+            self._publish_in_nav_msgs()
             self._pub.publish(trajs)
             self._publish_rate.sleep()
 
+    # add each traj to be published in nav_msg/Path
+    def _add_in_nav_msgs(self, uuid):
+        if uuid not in self.pub_nav:
+            rospy.loginfo("Creating a publisher for %s...", uuid)
+            name = uuid.replace("-", "0")
+            self.pub_nav[uuid] = rospy.Publisher(
+                self.name + '/' + name, Path, queue_size=10
+            )
```

Comment:
Did you try making that a latched topic so even if the person is not tracked any more, we could still visualise it?